admin/handlers.php

```diff
@@ -188,8 +193,11 @@ function gigpress_error_checking($context) {
 				$errors['artist_name'] = __("You must enter an artist name.", "gigpress");
 			if($_POST['show_venue_id'] == 'new' && empty($_POST['venue_name']))
 				$errors['venue_name'] = __("You must enter a venue name.", "gigpress");
-			if($_POST['show_venue_id'] == 'new' && empty($_POST['venue_city']))
-				$errors['venue_city'] = __("You must enter a city.", "gigpress");
+			if ( true === $venue_city_required ) {
+				if($_POST['show_venue_id'] == 'new' && empty($_POST['venue_city'])) {
```

Comment:
```suggestion
				if ( 'new' == $_POST['show_venue_id'] && empty( $_POST['venue_city'] ) ) {
```

Reply:
We normally follow Yoda conditionals.

Reply:
Good point!
That was original code which I didn't touch.